### ISSUE TITLE
babel compiler requires sourceMaps="both"?

### ISSUE THREAD

**Issue by boneskull** [wallabyjs/public#477]
### Issue description or question

If I do not have `sourceMaps: 'both'` in my babel compiler configuration, then I receive the following error in Wallaby console:

```
Cannot read property 'sections' of undefined
```

### .babelrc

```json
{
  "presets": [
    "es2015"
  ],
  "plugins": [
    "lodash"
  ],
  "sourceMaps": "inline",
  "env": {
    "jetbrains": {
      "retainLines": true
    }
  }
}
```

### Wallaby.js configuration file

```javascript
'use strict';

/* eslint import/no-require:0 */

module.exports = function wallabyConfig (wallaby) {
  var compiler = wallaby.compilers.babel({
    babelrc: true
  });

  return {
    files: [
      {
        pattern: 'node_modules/babel-polyfill/dist/polyfill.js',
        instrument: false
      },
      {
        pattern: 'src/**/*.js',
        load: false
      },
      {
        pattern: 'test/unit/fixture.js',
        load: false
      },
      {
        pattern: 'test/unit/**/*.spec.js',
        ignore: true
      },
      {
        pattern: 'package.json',
        load: false
      }
    ],
    tests: [
      {
        pattern: 'test/unit/**/*.spec.js',
        load: false
      }
    ],
    env: {
      runner: require('phantomjs-prebuilt').path,
      params: {
        runner: '--web-security=false'
      }
    },
    compilers: {
      'src/**/*.js': compiler,
      'test/unit/**/*.js': compiler
    },
    postprocessor: require('wallabify')({
      debug: true,
      entryPatterns: [
        'test/unit/fixture.js',
        'test/unit/**/*.spec.js'
      ]
    }),
    testFramework: 'mocha',
    bootstrap: function bootstrap () {
      window.__moduleBundler.loadTests();
    },
    debug: true
  };
};

```

### Code editor or IDE name and version

latest pycharm EAP

### OS name and version

latest OSX 

*debug log follows*

**Comment by boneskull** [comment]
[debug log](https://gist.github.com/boneskull/e0969d6f65e9e4332db2)

Of potential relevance, I'm consuming [source-map-support](https://www.npmjs.com/package/source-map-support).

**Comment by ArtemGovorov** [comment]
Wallaby sets the correct `sourceMap`  (without `s` at the end) value that it needs internally, but apparently when the `sourceMaps` (with `s` at the end) alias is used in `.babelrc`, it overrides the value that wallaby was setting. The issue is fixed in core v1.0.190.
